code/go/0chain.net/smartcontract/storagesc/benchmark_tests.go

```diff
@@ -522,13 +522,7 @@ func BenchmarkTests(
 				ToClientID:   ADDRESS,
 				CreationDate: creationTime,
 			},
-			input: func() []byte {
```

Comment:
probably we need empty struct here, have you ran benchmarks with these changes?

Reply:
check done.